### ISSUE TITLE
WDS PPT image

### ISSUE THREAD

**Issue by emilysaeli** [Microsoft/MCW-Microservices-architecture#9]
WDS PPT image, slide 17 (preferred solution) does not pass the color contrast test for accessibility standards.

**Comment by DawnmarieDesJardins** [comment]
More info...
White text on the light blue or grey images does not have enough contrast. to pass accessibility.  Suggest changing light blue to dark blue or making the text black and bigger.  We can't do it because we only have the flat image.

**Comment by michelebusta** [comment]
updated
